### PR TITLE
Update for WAR3 1.29 single hash file

### PR DESCRIPTION
**Pull request by Davnit**

### CODE REVIEW

**Comment by tagban** [comment]
Did this just drop?

**Comment by Davnit** [comment]
Yea the update came out on 4/10

File: src/Modules/Settings.rbbas [hunk 1]
```diff
@@ -192,10 +192,8 @@ Protected Module Settings
 		    If Base.Child("W3DM.bin") <> Nil Then Result(3) = Base.Child("W3DM.bin").AbsolutePath
 		    
 		  Case Packets.BNETProduct_WAR3, Packets.BNETProduct_W3XP
-		    If Base.Child("war3.exe") <> Nil Then Result(0) = Base.Child("war3.exe").AbsolutePath
-		    If Base.Child("Storm.dll") <> Nil Then Result(1) = Base.Child("Storm.dll").AbsolutePath
-		    If Base.Child("game.dll") <> Nil Then Result(2) = Base.Child("game.dll").AbsolutePath
-		    Result(3) = ""
+		    If Base.Child("Warcraft III.exe") <> Nil Then Result(0) = Base.Child("Warcraft III.exe").AbsolutePath
+		    Result(1) = ""
```
Comment (carlbennett) [inline src/Modules/Settings.rbbas:196]: The last element in the array is not required. It was intended to be the `WAR3.bin` file, but it's not needed.

**Review by carlbennett — CHANGES_REQUESTED** [review]

**Comment by carlbennett** [comment]
Going to merge and fix upstream.

File: src/Modules/Settings.rbbas (hunk 1 above)
Comment (Davnit) [reply src/Modules/Settings.rbbas:196]: Ah I was just copying the D2 fix from 036429a

**Comment by carlbennett** [comment]
Fixed upstream in 1426a6de955d6232386c86a228b6a6cfa3525e09.

**Pull request merged**